11.LoDSystem/main.cpp

```diff
@@ -686,21 +704,23 @@ class LoDSystemApp : public ApplicationBase
                         auto& drawCallOffsetsInDWORDs = lodLibraryData.drawCallOffsetsIn20ByteStrides;
                         for (auto i = 0u; i < cullingParams.drawcallCount; i++)
                             drawCallOffsetsInDWORDs[i] = lodLibraryData.drawCallOffsetsIn20ByteStrides[i] * sizeof(asset::DrawElementsIndirectCommand_t) / sizeof(uint32_t);
+                        video::IGPUBuffer::SCreationParams p1; p1.size = cullingParams.drawcallCount * sizeof(uint32_t);
+                        video::IGPUBuffer::SCreationParams p2; p2.size = lodLibraryData.drawCountOffsets.size() * sizeof(uint32_t);
                         cullingParams.transientInputDS = culling_system_t::createInputDescriptorSet(
                             logicalDevice.get(), cullingDSPool.get(),
                             culling_system_t::createInputDescriptorSetLayout(logicalDevice.get()),
                             cullingParams.indirectDispatchParams,
                             cullingParams.instanceList,
                             cullingParams.scratchBufferRanges,
-                            { 0ull,~0ull,utilities->createFilledDeviceLocalBufferOnDedMem(transferUpQueue,cullingParams.drawcallCount * sizeof(uint32_t),drawCallOffsetsInDWORDs.data()) },
-                            { 0ull,~0ull,utilities->createFilledDeviceLocalBufferOnDedMem(transferUpQueue,lodLibraryData.drawCountOffsets.size() * sizeof(uint32_t),lodLibraryData.drawCountOffsets.data()) }
+                            { 0ull,~0ull,utilities->createFilledDeviceLocalBufferOnDedMem(transferUpQueue, std::move(p1), drawCallOffsetsInDWORDs.data()) },
```

Comment:
Please name these creation params correctly, not `p1` and `p2`
See `culling_system_t::createInputDescriptorSet` function to see what these params are named